pkg/kubelet/cm/dra/manager.go

```diff
@@ -68,22 +68,35 @@ func NewManagerImpl(kubeClient clientset.Interface, stateFileDirectory string) (
 func (m *ManagerImpl) PrepareResources(pod *v1.Pod) error {
 	for i := range pod.Spec.ResourceClaims {
 		podClaim := &pod.Spec.ResourceClaims[i]
-		claimName := resourceclaim.Name(pod, podClaim)
-		klog.V(3).InfoS("Processing resource", "claim", claimName, "pod", pod.Name)
+		klog.V(3).InfoS("Processing resource", "podClaim", podClaim.Name, "pod", pod.Name)
+		claimName, mustCheckOwner, err := resourceclaim.Name(pod, podClaim)
+		if err != nil {
+			return err
```

Comment:
We should probably wrap this error to be consistent.

Reply:
Okay.